sdk/formrecognizer/Azure.AI.FormRecognizer/src/GeneratorStubs/PublicTypes.cs

```diff
@@ -15,6 +15,9 @@
 
 namespace Azure.AI.FormRecognizer.DocumentAnalysis
 {
+    [CodeGenModel("DocumentAnalysisFeature")]
+    public partial struct DocumentAnalysisFeature { }
```

Comment:
For some reason CodeGen is putting this struct in the wrong namespace (`Azure.AI.FormRecognizer.Models` instead of `Azure.AI.FormRecognizer.DocumentAnalysis`). It's probably related to the fact that it's an input type.

We can overwrite this behavior by adding it to this class (the `CodeGenModel` attribute is necessary in this case, even though we're not renaming it).